<launch>

<include file="$(find usb_cam)/launch/usb_cam-test.launch" />

<node name="optic_flow_node" pkg="optic_flow_node" type="wide_field_optic_flow_node" output="screen" >
	<param name="image_center_x" value="290"/>
	<param name="image_center_y" value="240"/>
	<param name="inner_ring_radius" value="155"/>
	<param name="ring_dr" value="20"/>
	<param name="num_ring_points" value="80"/>
	<param name="num_rings" value="5"/>
</node>

<include file="/opt/ros/melodic/share/vrpn_client_ros/launch/optitrack.launch" />

<node pkg="vrpn_velocity" type="vrpn_vel_calc.py" name="vrpn_vel_calc" /> 

<node name="simple_depth" pkg="depth_estimation" type="simple_depth_node" output="screen" >
	<param name="num_ring_points" value="80"/>
</node>

</launch>
